### DISCUSSION TITLE
Trying to Create a Running Stitch Between 2 Fills

### DISCUSSION THREAD

**Discussion by st11x** [inkstitch/inkstitch#2035] [Q&A]
Hi,

Instead of a jump, I am trying to create a running stitch between 2 fills of the same color. This will be hidden by an upper layer.

I have this set up.

![image](https://user-images.githubusercontent.com/7398499/215342130-c893d418-4d98-4846-8abb-94d999296831.png)

Sometimes it works, sometimes I don't need the start positions, and sometimes the start positions are ignored. If I try to line up the start of the running stitch with the start of the next fill stitch, it has a higher chance of success, but it is very tricky to do so.

Is there a simpler or foolproof way to do this?

Here's a short clip of the render. It actually made it worse with the 2 jumps.

https://user-images.githubusercontent.com/7398499/215342141-5befe8de-7ba6-47df-a63c-5d4c7d22a3be.mp4

Thanks

Matt

**Comment by kaalleen** [comment]
Select your running stitch and run "Path > Reverse Path" on it. It has the wrong direction.

**Reply by st11x** [reply]
I tried that too but it didn't work. I am attaching a simple example that looks like it should work but doesn't.

![jump](https://user-images.githubusercontent.com/7398499/215350077-d969db4e-e9c9-4b61-92fb-1113fbaa5539.svg)
